src/redux/slices/auth.js

```diff
@@ -23,6 +23,7 @@ export const authSlice = createSlice({
     },
     Logout: () => {
       localStorage.removeItem('user');
+      return null;
```

Comment:
Good Job! I have a clarification question in here, what's the reason of returning null? why do we need this?